docs/articles/commands/processors/slash_commands/translator_providers.md

```diff
@@ -0,0 +1,55 @@
+---
+uid: articles.commands.command_processors.slash_commands.translator_providers
+title: Translator Providers
+---
+
+# Translator Providers
+
+In the event that you would like to provide translations for your commands, you can use the `ITranslatorProvider` interface. This interface allows you to provide translations for your commands and parameters. Here is the interface:
```

Comment:
surely you meant to name this "Interaction Localizer"? the rest of the article is consistent, it's just these few lines

Reply:
Sure did. Copy and pasted one too many times.